servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java

```diff
@@ -171,7 +309,41 @@ public <T> Publisher<? extends T> transportEventStream(final HttpEventKey<T> eve
 
         @Override
         public Single<StreamingHttpResponse> request(final StreamingHttpRequest request) {
-            return delegate.request(request);
+            return Single.defer(() -> {
```

Comment:
Btw, if you don't need public API for error classifiers at the first iteration, there might be even better place for this logic. Consider `LoadBalancedStreamingHttpClient` that already uses `BeforeFinallyHttpOperator` that you can take advantage of and there is also no need for `defer`.

Reply:
Good idea, but the new APIs (which were removed for this round) will more likely than not, be added in the near future. Which means there will be wiring involved to get these into that layer. Therefore, I chose to keep everything in the current placement.